roles/tendrl-performance-monitoring/tasks/main.yml

```diff
@@ -21,8 +21,33 @@
   with_items:
     - regexp: '^etcd_connection:*'
       line: "etcd_connection{{ ':' }} {{ groups['usm_server'][0] }}"
-    - regexp: '^api_server_addr:*'
-      line: "api_server_addr{{ ':' }} {{ groups['usm_server'][0] }}"
+    - regexp: '^etcd_port:*'
+      line: "etcd_port{{ ':' }} 2379"
+    - regexp: '^time_series_db_server:*'
+      line: "time_series_db_server{{ ':' }} {{ ansible_eth0['ipv4']['address'] }}"
+    - regexp: '^time_series_db_port:*'
+      line: "time_series_db_port{{ ':' }} 10080"
+
+- name: Init graphite-db
+  command: /usr/lib/python2.7/site-packages/graphite/manage.py syncdb --noinput
```

Comment:
Could you add when statement so that this is not done again when the db file already exists?